x/oracle/keeper/aggregator/aggregator.go

```diff
@@ -48,6 +48,34 @@ type aggregator struct {
 	dsPrices map[uint64]string
 }
 
+func (agg *aggregator) copy4CheckTx() *aggregator {
+	ret := &aggregator{
+		finalPrice:  big.NewInt(agg.finalPrice.Int64()),
```

Comment:
If agg.finalPrice exceeds the range of Int64, an integer overflow will occur. It is recommended to copy the value directly using the Set method of big.Int like:
`finalPrice: big.NewInt(0).Set(agg.finalPrice),
`

Reply:
fixed